spec/firebase_app_distribution_action_spec.rb

```diff
@@ -1,141 +1,214 @@
 describe Fastlane::Actions::FirebaseAppDistributionAction do
   let(:fake_file) { StringIO.new }
-
-  before do
+  let(:temp_connection) { double("Connection") }
+  # These examples are copied directly from API responses. Maybe we should move them to another file?
+  let(:fake_get_app_response_valid) { double("Response", status: 200, body: { projectNumber: "1", appId: "1", platform: "android", bundleId: "1", contactEmail: "Hello@world.com" }) }
```

Comment:
I love all these test cases! Can you format them so they're a little easier to read?

Maybe something like:
```suggestion
  let(:fake_get_app_response_valid) do
    double("Response", status: 200, body: {
      projectNumber: "1",
      ...,
      ...,
    })
end
```

Reply:
Also can you use different values for the app, bundle, and project IDs, so the person reading the test can see what's what? Feel free to use easy to read values like "test-project-id" or "fake-bundle-id"